### ISSUE TITLE
Can't delete a minion key from the Python API

### ISSUE THREAD

**Issue by whiteinge** [saltstack/salt#31112]
This doesn't work. No error, no return. Tested on 2015.8 and 2016.3.

```python
>>> import salt.wheel
>>> import salt.config
>>> master_opts = salt.config.client_config('/etc/salt/master')
>>> wheel = salt.wheel.WheelClient(master_opts)
>>> wheel.cmd('key.delete', kwarg={'match': 'jerry'})
```

**Comment by rallytime** [comment]
@whiteinge I think this might be a duplicate of #30866. What do you think?

**Comment by rallytime** [comment]
@whiteinge I think this should be fixed with #31570.

**Comment by cro** [comment]
verified fixed.
